react-client-app/src/components/UploadScanReport.jsx

```diff
@@ -53,7 +53,7 @@ const UploadScanReport = ({ setTitle }) => {
         setLoadingDataset(true)
         const dataPartnerId = selectedDataPartner.id
         if (dataPartnerId != undefined) {
-            const datasets_query = await useGet(`/datasets/?data_partner=${dataPartnerId}`)
+            const datasets_query = await useGet(`/datasets/?data_partner=${dataPartnerId}&hidden=false`)
```

Comment:
```suggestion
            // Show only Datasets that have this Data Partner and are no archived
            const datasets_query = await useGet(`/datasets/?data_partner=${dataPartnerId}&hidden=false`)
```
Just being a bit more explicit